config.py

```diff
@@ -392,6 +398,35 @@ def get_config(parse_args=True, cfg_path=None, options=None):
         else:
             agentConfig['use_dogstatsd'] = True
 
+        # Service discovery
+        if config.has_option('Main', 'service_discovery_backend'):
```

Comment:
This block should be located in something like utils.service_discovery.config or something, let's not add too much stuff to this already big module